libs/agno/agno/workflow/workflow.py

```diff
@@ -169,6 +169,11 @@ class Workflow:
     # This helps us improve the Agent and provide better support
     telemetry: bool = True
 
+    # Add this flag to control if the workflow should add history to the steps
+    add_workflow_history: bool = False
```

Comment:
too ambiguous..?

`add_history_to_steps` at `Workflow` level and `add_history` at `Step` level